vita3k/gui/src/settings_dialog.cpp

```diff
@@ -424,16 +424,16 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         ImGui::Spacing();
         static const char *LIST_CPU_BACKEND[] = { "Dynarmic", "Unicorn" };
         static const char *LIST_CPU_BACKEND_DISPLAY[] = { "Dynarmic", "Unicorn (deprecated)" };
-        ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "Cpu Backend");
+        ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "CPU Backend");
         if (ImGui::Combo("##cpu_backend", reinterpret_cast<int *>(&config_cpu_backend), LIST_CPU_BACKEND_DISPLAY, IM_ARRAYSIZE(LIST_CPU_BACKEND_DISPLAY)))
             config.cpu_backend = LIST_CPU_BACKEND[int(config_cpu_backend)];
         if (ImGui::IsItemHovered())
-            ImGui::SetTooltip("Select your preferred cpu backend.");
+            ImGui::SetTooltip("Select your preferred CPU backend.");
         if (config_cpu_backend == CPUBackend::Dynarmic) {
             ImGui::Spacing();
             ImGui::Checkbox("Enable optimizations", &config.cpu_opt);
             if (ImGui::IsItemHovered())
-                ImGui::SetTooltip("Enable additional CPU JIT optimizations.");
+                ImGui::SetTooltip("Enables additional CPU JIT optimizations.");
```

Comment:
Why S ?

Reply:
this sentence literally doesn't mean the same thing anymore
when you adding S it no said "activate additonal" but allow additionnal, is totaly wrong